go/twofer: comments for disapprove (#1222)
exercism/website-copy: automated-comments/go/two-fer/strings_join_used_for_concatenation.md

> `strings.Join` is perfect if the data already comes in the form of a `[]string`.

Reviewer: ```suggestion
Don't use `strings.Join` if the data is not of type `[]string`. 
```

I suggest being more stern here. Right now it's a statement where I can respond with "okay, so is it wrong now??". If you phrase it like the suggestion here, it's instantly clear(er).

Reviewer 2: Notwithstanding the specific advice here, which I think is right, I'm not sure 'stern' is exactly the right word for the tone we want. Clear, concise, and easy to act on, yes. But can't we be all those things and still be gentle, polite, encouraging, and collegial?

Reviewer: No. The analyzer should not spit out _emotive_ content. It should sound neutral (hence me using "stern").

Reviewer 2: Understood. I think this probably exposes a pretty fundamental difference of approach; I'm not sure there's anything more I can usefully contribute to the automated analyzer as a project, if that's the case. We'd just end up arguing over the same points for every comment.

Reviewer: Does [this](https://github.com/exercism/automated-mentoring-support/blob/5b01bae1faced4c771308bab10a4fd6277e2a4d1/docs/comment_copy_guidelines.md) perhaps help?

Reviewer: (But hey, I'm not the authority here! Just following what we came up with earlier 😅).

Let's also ask @iHiD and @kytrinyx from a product/platform point of view.

Author: ```suggestion
Don't use `strings.Join` if the data is not of type `[]string`.
```

> Take a look at `fmt.Sprintf` for this use case; it's a very powerful way of building up strings from multiple pieces.

Reviewer: ```suggestion
Use `fmt.Sprintf` instead it's a very powerful way of building up strings from multiple pieces.
```

Author: ```suggestion
Use `fmt.Sprintf` instead. It's a very powerful way of building up strings from multiple pieces.
```

> 

Author: ```suggestion
```

> In case of simple concatenations, the `+` operator is a very fast alternative.

Reviewer: I suggest removing this. When the student uses Sprintf, it will be given as a comment on line 5 of [automated-comments/go/two-fer/strings_builder_used_for_concatenation.md](https://github.com/exercism/website-copy/pull/1222/files#diff-da6983c41277b728eff789945e1250e4).

```suggestion
```
